rollbar/__init__.py

```diff
@@ -1217,6 +1229,7 @@ def _parse_response(path, access_token, params, resp, endpoint=None):
         return
     elif resp.status_code == 413:
         log.warning("Rollbar: request entity too large. Payload was: %r", params)
+        _send_failsafe('Payload too large')
```

Comment:
Can we make this a litttle more descriptive and useful? Customers are going to see this, not know what it is, and ask support. How about we change this text to:

```
Failsafe from pyrollbar: payload too large. Original payload may be found in your server logs by searching for the UUID.
```

and attach the following as metadata to the failsafe payload:

- the UUID (first 255 characters, just to be safe)
- the `server.host` value (first 255 characters, just to be safe)

Then, for the log message on line 1231, include the UUID at the beginning of the line:

```
log.warning("Rollbar: request entity too large for UUID %r\nPayload:\n%r", uuid, params)
```